### PR TITLE
PORTS -- Cloudy

### PR DESCRIPTION
**Pull request by CloudyLopez**
# JS Adagrams
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
--- | ---
What patterns were you able to use from your Ruby knowledge to apply to JavaScript? | methods and loops are pretty similar.
Did you need to use different strategies to find information online about JavaScript than you would for Ruby? | yes, the mdn is the go to place but sometimes feels sparser then info on ruby.
What was a challenge you were able to overcome on this assignment? | trying to figure out how to do things when shuffle and inject are not available
What has been interesting and positive about learning a new programming language? | seeing the patterns and understanding the syntax is everything 
What is something to focus on your learnings in JavaScript in the next week? | get a great understanding of syntax and getting into the details.

### CODE REVIEW

File: src/adagrams.js [hunk 1]
```diff
@@ -1,8 +1,65 @@
...
-  drawLetters() {
-    // Implement this method for wave 1
+  drawLetters() { //ln 3
+    const poolOfLetters = [
+      "A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"
+    ];
+    let myLetters = []; //ln 7
+    for (let i = 0; i < 10; i +=1) {
+      let randomIndexNum = Math.floor(Math.random() * poolOfLetters.length); //shuffle isnt real //using length for index
+      myLetters.push(poolOfLetters[randomIndexNum]);
```
Comment (CheezItMan) [inline src/adagrams.js:9]: How do you prevent the same index from being selected multiple times?

**Comment by CheezItMan** [comment]
# JS Adagrams
## What We're Looking For



Feature | Feedback
--- | ---
**General** |
Answered comprehension questions | Check, `inject` is available in Ruby, however it's called `reduce`.  
Small commits with meaningful commit messages | Check
**Code Requirements** |
`drawLetters` method | Check
Uses appropriate data structure to store the letter distribution | Check
All tests for `drawLetters` pass | Check
`usesAvailableLetters` method | Check
All tests for `usesAvailableLetters` pass | Check
`scoreWord` method | Check
Uses appropriate data structure to store the letter scores | Check
All tests for `scoreWord` pass | Check
`highestScoreFrom` method | Check
Appropriately handles edge cases for tie-breaking logic | Check
All tests for `highestScoreFrom` pass | Check
**Overall** |  Nice work you hit the learning goals here for this project.  Do see my inline notes and let me know if you have questions.
